### PR TITLE
adds link to search object type filter

### LINKED ISSUE #150: Facet values in body should link back to search results page
- MATERIALS
- TYPE
- CATEGORY

### PR DESCRIPTION
**Pull request by Danwhy**
Adds link from resource page to search results with 'object type' filter.

Last facet link from resource page, so closes #150

### CODE REVIEW

**Comment by alanshaw** [comment]
👓

File: lib/transforms/json-to-html-data.js [hunk 1]
```diff
@@ -328,15 +328,15 @@ function getNotes (data) {
...
+function getObjectType (data, links) {
   var value;
   if (data.type !== 'people' && data.attributes.name) {
     data.attributes.name.forEach(el => {
       if (el.primary) {
         value = el.value || null;
       }
     });
-    return value ? {key: 'type', value: value} : null;
+    return value ? {key: 'type', value: value, link: links.root + '/search/objects?filter[type]=' + value} : null;
```
Comment (alanshaw) [inline lib/transforms/json-to-html-data.js]: `value` in URL needs `encodeURIComponent` in case it contains URI reserved characters like & for example.

Comment (Danwhy) [inline lib/transforms/json-to-html-data.js]: 👍 I did that for the other links, will add it to this one

**Comment by alanshaw** [comment]
LGTM

**Pull request merged**